src/lemma.js

```diff
@@ -127,6 +127,19 @@ class Lemma {
 
     this.translation = translation
   }
+
+  /**
+   * Test to see if two lemmas could represent the same word
+   * @param {Lemma} lemma the lemma to compare
+   * @return {Boolean} true or false
+   */
+  isEqual (lemma) {
```

Comment:
If I understand it correctly, it does check that word and part of speech are the same only. It does not check for full feature equality. So I'm thinking that `isEqual` name might be confusing in this case. Maybe it's better to rename it to something that more fully reflects a partiality of lemma comparison (and we might want to use `isEqual` later for full feature comparison)? What do you think?

Reply:
@kirlat Yes, that's a good point. Is there a good naming convention here that you can suggest? I could use 'matches' or 'mayEqual' but if there is a more standard way to represent this I'd like to use it.

Reply:
I think it's good if name would reflect an essence of a comparison. So if we're comparing a word and POFS what do really want to know about being equal between the two (from business logic standpoint)? Is it a word? Maybe we can call it `matchesWord` or `matchesWordPOFS`, or maybe `isSameWord` or maybe `isSameWordPOFS`.

It's really not easy. I think naming is probably the hardest part of software development :-).

Reply:
I did a little searching and apparently linguistically, this scenario is known as a "full homonym" - two words spelled the same with the same part of speech. Since that's what I'm testing here I will name the method accordingly - i.e. isFullHomonym . Sound good?

Reply:
Yes, I think `isFullHomonym` sounds perfect!